aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -1,4 +1,5 @@
-from __future__ import print_function
+#! /usr/bin/env python3
```

Comment:
There isn't usually a space before the interpreter path

Reply:
:+1: Fixed